productized/logic/build-quarkus-app.sh

```diff
@@ -42,7 +42,7 @@ maven_plugins_gav=("org.apache.maven.plugins:maven-resources-plugin:3.3.1" "org.
 properties_with_versions=("compiler-plugin.version:3.11.0" "surefire-plugin.version:3.1.2")
 
 # extra dependencies required for offline mode
-quarkus_extensions_extra_deps="com.aayushatharva.brotli4j:native-linux-aarch64:1.12.0"
+quarkus_extensions_extra_deps="com.aayushatharva.brotli4j:native-linux-aarch64:1.16.0.redhat-00003"
```

Comment:
For product build we use `com.redhat.quarkus.platform:quarkus-bom` and the RHBQ 3.8.4 will be prod version `3.8.4.redhat-00002` (to be released soon).

So this productized platform quarkus-bom has brotli4j1.16.0.redhat-00003. For more info why we do this see KOGITO-9565